src/main/plugin/iso19139.ca.HNAP/loc/eng/schematron-rules-multilingual.xml

```diff
@@ -45,7 +45,8 @@
   <SecurityNoteMismatchedBothLang>Value mismatched for Security User Note in both languages</SecurityNoteMismatchedBothLang>
   <SecurityClassificationUserNote>Security User Note is not valid for the classification code selected. Valid values are:</SecurityClassificationUserNote>
   <SecurityClassificationUserNoteEmpty>Security User Note should be empty for the classification code selected</SecurityClassificationUserNoteEmpty>
-  <OtherConstraintsNote>If you indicate 'Other Restrictions' in the 'Access Constraints' or 'Use Constraints' fields, the other constraints for accessing or using the resource should be explained here.</OtherConstraintsNote>
+  <OtherConstraintsNote>If you have indicated 'Other Restrictions' in either the 'Access Constraints' or 'Use Constraints' fields, the ‘Other constraints' text box must be completed.</OtherConstraintsNote>
```

Comment:
Should " text box must be completed" changed to " text box must be filled".

I don't believe completed


If you have indicated 'Other Restrictions' in either the 'Access Constraints' or 'Use Constraints' fields, ensure to provide information for 'Other Constraints' field.